hack/latest-binaries.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ "$#" -ne 1 ]; then
+  echo "usage: $0 KUBERNETES_MINOR_VERSION"
+  exit 1
+fi
+
+MINOR_VERSION="${1}"
+
+LATEST_BINARIES=$(aws s3api list-objects-v2 --bucket amazon-eks --prefix "${MINOR_VERSION}" --query 'Contents[*].[Key]' --output text | cut -d'/' -f-2 | sort -V -r | uniq | head -n1)
```

Comment:
Can provide a comment here that explains what this is doing and what the directory should look like?